robot_skills/src/robot_skills/util/kdl_conversions.py

```diff
@@ -35,6 +35,23 @@ def extractVectorStamped(self):
         return VectorStamped(frame_id=self.frame_id, vector=self.frame.p)
 
 
+def frame_stamped(frame_id, x, y, z, roll=0, pitch=0, yaw=0):
```

Comment:
There already is a FrameStampedFromXYZRPY on the bottom of this file. Granted, this function is easier on the eyes. 

I really need to get rid of this CamelCase bug in my head